### PR TITLE
Intervention

### PR DESCRIPTION
**Pull request by vandevoorde**
First working version of attemptGoal.

### CODE REVIEW

File: Analytics/Classes/SEGAnalytics.h [hunk 1]
```diff
@@ -110,6 +110,32 @@ NS_ASSUME_NONNULL_BEGIN
...
 - (void)screen:(NSString *)screenTitle;
 
+typedef void(^SEGGoalSuccessCallback)(NSString * _Nullable variation);
+typedef void(^SEGGoalFailureCallback)(void);
+
+/*!
+ * @method
+ *
+ * @abstract
+ * Query if the application should attempt to acheive a particular goal, e.g., email collection.
```
Comment (quirkyllama) [inline Analytics/Classes/SEGAnalytics.h]: achieve

File: Examples/ManualExample/ManualExample/Assets.xcassets/AppIcon.appiconset/Contents.json [hunk 2]
```diff
@@ -1,5 +1,15 @@
 {
   "images" : [
+    {
```
Comment (quirkyllama) [inline Examples/ManualExample/ManualExample/Assets.xcassets/AppIcon.appiconset/Contents.json:3]: What do these resource changes do?

**Review by quirkyllama — APPROVED** [review]

File: Examples/ManualExample/ManualExample/Assets.xcassets/AppIcon.appiconset/Contents.json (hunk 2 above)
Comment (vandevoorde) [reply Examples/ManualExample/ManualExample/Assets.xcassets/AppIcon.appiconset/Contents.json:3]: This is something Xcode must have done, and I'm not sure what triggered it.
Perhaps I should revert this part and see if anything breaks.

Comment (vandevoorde) [reply Examples/ManualExample/ManualExample/Assets.xcassets/AppIcon.appiconset/Contents.json:3]: FTR, if I undo the change, Xcode seems to redo it instantly.

**Pull request merged**